supersetapiclient/charts.py

```diff
@@ -11,6 +11,7 @@ class Chart(Object):
     JSON_FIELDS = [
         "params"
     ]
+    LIST_OF_OBJECT_FIELDS = {}
```

Comment:
`LIST_OF_OBJECT_FIELDS = {}` on child classes of `Object` are unnecessary because it is already available via the parent class.

Reply:
Thanks, I've removed the redundant declarations.